flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/sink/BaseSink.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2024 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.flink.bigquery.sink;
+
+import org.apache.flink.api.connector.sink2.Sink;
+
+/** Base class for developing a BigQuery sink. */
+abstract class BaseSink implements Sink {}
```

Comment:
assuming this will be the base implementation that is shared by at-least-once and exactly-once implementation. if so, can we name this BigQuerySink or BigQueryBaseSink? (same with the file name)